packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -371,6 +372,8 @@ class SkiaGoldClient {
     return json.encode(
       <String, dynamic>{
         'Platform' : platform.operatingSystem,
+        if (platform.environment[_kTestBrowserKey] != null)
```

Comment:
The documentation above will need to be updated.

Reply:
Done.